src/utils/i18n.js

```diff
@@ -62,10 +62,22 @@ function hget(d, key, defaultValue){
 
 export function t(trans, lang, key, ...params){
     var kl = key
+    var fb
     if (!Array.isArray(kl))
         kl = [kl]
-    const value = hget(trans, [lang, ...kl])
-    if (value === undefined){
+    var value = hget(trans, [lang, ...kl])
+
+    // try to get first language as fallback
```

Comment:
This comment might be outdated already as the code seems to look through all translations to find a fallback.

Reply:
But is does exit on the first match :)

Reply:
https://github.com/mkkeck/coi/commit/bf9bcd4cf5546bdebd8344c377c6ccb18fc7644b